README.md

```diff
@@ -128,17 +129,33 @@ In addition, the provider can be configured using attributes in the
 configuration. Attributes specified in `main.tf` override the configuration loaded by
 `conjur-api-go`.
 
-For example, if the environment is initialized as above, this configuration would
-authenticate as `terraform-user` instead of `admin`:
+For example, with `conjur_api_key` and `conjur_ssl_cert`defined as
+[input variables](https://www.terraform.io/docs/configuration/variables.html), this
+type of configuration could be used:
 
 ```
 # main.tf
+variable "conjur_api_key" {}
+variable "conjur_ssl_cert" {}
+# If you have the certificate as a file, use this line instead
+# variable "conjur_ssl_cert_path" {}
+
 provider "conjur" {
-  login = "terraform-user"
-  api_key = "x0dwqc3jrqkye3xhn7k62rw31c6216ewfe1wv71291jrqm4j15b3dg9"
+  appliance_url = "http://conjur-server"
+  ssl_cert = var.conjur_ssl_cert
+  # If you have the certificate as a file, use this line instead
+  # ssl_cert_path = var.conjur_ssl_cert_path
+
+  account = "myorg"
+
+  login = "admin"
+  api_key = var.conjur_api_key
 }
 ```
 
+_Note as well that `login`/`api_key` settings are only used when both are present in
+the attributes otherwise the provider will fall back to using environment variables
+for those two values_
```

Comment:
Can we change this to something like

> **Notes on precedence of configuration variable setting:**
> - If both the environment variable **and** `.tf` configuration are present for a configuration setting, the `.tf` configuration takes precedence and the environment variable will be ignored.
> - If the `.tf` configuration does not include **both** `login` and `api_key`, then environment variables will be used for these values instead.

is there anything else worth noting here?

Reply:
sounds good. the login/api_key is the only weird thing there.